diff --git a/langs/ko_KR.js b/langs/ko_KR.js
index ce0e42c..631e734 100644
--- a/langs/ko_KR.js
+++ b/langs/ko_KR.js
@@ -194,8 +194,8 @@ tinymce.addI18n('ko_KR',{
 "Table properties": "\ud14c\uc774\ube14 \uc18d\uc131",
 "Delete table": "\ud14c\uc774\ube14 \uc0ad\uc81c",
 "Cell": "\uc140",
-"Row": "\uc5f4",
-"Column": "\ud589",
+"Row": "\ud589",
+"Column": "\uc5f4",
 "Cell properties": "\uc140 \uc18d",
 "Merge cells": "\uc140 \ud569\uce58\uae30",
 "Split cell": "\uc140 \ub098\ub204\uae30",
@@ -207,7 +207,7 @@ tinymce.addI18n('ko_KR',{
 "Copy row": "\ud589 \ubcf5\uc0ac",
 "Paste row before": "\uc774\uc804\uc5d0 \ud589 \ubd99\uc5ec\ub123\uae30",
 "Paste row after": "\ub2e4\uc74c\uc5d0 \ud589 \ubd99\uc5ec\ub123\uae30",
-"Insert column before": "\uc774\uc804\uc5d0 \ud589 \uc0bd\uc785",
+"Insert column before": "\uc774\uc804\uc5d0 \uc5f4 \uc0bd\uc785",
 "Insert column after": "\ub2e4\uc74c\uc5d0 \uc5f4 \uc0bd\uc785",
 "Delete column": "\uc5f4 \uc9c0\uc6b0\uae30",
 "Cols": "\uc5f4",
